edx_proctoring/backends/software_secure.py

```diff
@@ -390,3 +392,23 @@ def get_video_review_aes_key(self):
         Returns the aes key used to encrypt the video review url
         """
         return self.video_review_aes_key
+
+    def get_instructor_url(
+        self, course_id, user, exam_id=None, attempt_id=None,
+        show_configuration_dashboard=False, encrypted_video_review_url=None
+    ):
+        """
+        Returns the url for video reviews
+        """
+        # video_review_url is required for PSI backend
+        if not encrypted_video_review_url:
+            return None
+
+        aes_key = codecs.decode(self.video_review_aes_key, "hex")
```

Comment:
Can you implement a try and catch here?
If the decryption is erroring out, please log it, return `none`. I think we shouldn't let a bad decrypt to cause 500 errors.

Reply:
Also, could you test the error handling on bad decrypt with unit tests, if possible?